### PR TITLE
feat: add quiz app unit tests

### PR DESCRIPTION
**Pull request by MADAkwasi**

### CODE REVIEW

File: week_6/testing/src/app/header/header.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    component = fixture.componentInstance;
+
+    fixture.detectChanges();
+  });
+
+  it('renders correctly', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('Switches between light and dark mode', () => {
```
Comment (fredrickyalley) [inline week_6/testing/src/app/header/header.component.spec.ts:27]: start a description with small letter. use 'should switch between light and dark mode'

**Comment by fredrickyalley** [comment]
Add test for these components as well,
result, answer and question

File: week_6/testing/src/app/header/header.component.spec.ts (hunk 1 above)
Comment (MADAkwasi) [reply week_6/testing/src/app/header/header.component.spec.ts]: done

**Comment by MADAkwasi** [comment]
> Add test for these components as well, result, answer and question

done

**Review by fredrickyalley — APPROVED** [review]

**Pull request merged**